basic_module/account/src/traits/impls.rs

```diff
@@ -0,0 +1,117 @@
+use super::core::handles::*;
+use crate::account::{add_balance, get_balance, get_sequence, sub_balance};
+use crate::{get_context, get_storage_id, Account, Action};
+use key::verify_ed25519;
+
+#[allow(dead_code)]
+pub struct Handler {}
+
+impl CheckTxHandler for Handler {
+    fn verify_transaction(tx: &Transaction) -> bool {
+        match &tx.action {
+            Action::Pay {
+                sender,
+                receiver: _,
+                quantity,
+            } => {
+                let sender_balance = get_balance(&sender);
+                let sender_sequence = get_sequence(&sender);
+                match (sender_balance, sender_sequence) {
+                    (Some(balance), Some(sequence)) => !(balance < tx.fee + quantity || sequence <= tx.seq),
+                    _ => false,
+                }
+            }
+            _ => panic!("This is a wrong transaction format that the module account couldn't handle."),
+        }
+    }
+}
+
+#[allow(dead_code)]
+pub struct Executor {}
+
+impl TransactionExecutor for Executor {
+    fn execute_transactions(transactions: &[Transaction]) {
+        for tx in transactions {
+            match &tx.action {
+                Action::Pay {
+                    sender,
+                    receiver,
+                    quantity,
+                } => {
+                    let _ = sub_balance(&sender, tx.fee + quantity).and_then(|_| add_balance(&receiver, *quantity));
+                }
+                Action::Custom {
+                    ..
+                } => {}
+            }
+        }
+    }
+}
+
+#[allow(dead_code)]
+pub struct Manager {}
+
+impl AccountManager for Manager {
+    fn add_balance(address: &Address, val: u64) -> Result<(), AccountError> {
+        add_balance(address, val)
```

Comment:
Someone can misunderstand this as a recursive call so that let's handle this issue with `use ${path_to_add_balance}::add_balance as core_add_balance`